packages/localize-tools/src/modes/transform.ts

```diff
@@ -498,25 +501,15 @@ class Transformer {
 
   /**
    * Return whether the given symbol looks like one of the lit-localize modules
-   * (because it exports one of the special tagged functions).
+   * based on its filename. Note when we call this function, we're already
+   * strongly suspecting a lit-localize call.
    */
-  isLitLocalizeModule(moduleSymbol: ts.Symbol): boolean {
-    if (!moduleSymbol.exports) {
-      return false;
-    }
-    const exports = moduleSymbol.exports.values();
-    for (const xport of exports as typeof exports & {
-      [Symbol.iterator](): Iterator<ts.Symbol>;
-    }) {
-      const type = this.typeChecker.getTypeAtLocation(xport.valueDeclaration);
-      const props = this.typeChecker.getPropertiesOfType(type);
+  fileNameAppearsToBeLitLocalize(moduleSymbol: ts.Symbol): boolean {
+    for (const decl of moduleSymbol.declarations) {
       if (
-        props.some(
-          (prop) =>
-            prop.escapedName === '_LIT_LOCALIZE_MSG_' ||
-            prop.escapedName === '_LIT_LOCALIZE_CONTROLLER_FN_' ||
-            prop.escapedName === '_LIT_LOCALIZE_DECORATOR_'
-        )
+        ts.isSourceFile(decl) &&
+        (decl.fileName.endsWith('/localize/lit-localize.d.ts') ||
```

Comment:
Note to self:

We also need to check for the new `/init/transform.d.ts` file so that if the user imports it that way (which I think will become the required method in an upcoming breaking change), we will remove the import.

We should really have a universal way to notice "any import from the localize package". Maybe we just look for `@lit/localize` in the path -- but this doesn't work for tests because we import directly with a relative path (but maybe we can add some special handling for that).